pyvista/plotting/widgets.py

```diff
@@ -2629,6 +2633,77 @@ def clear_button_widgets(self):
         """Remove all of the button widgets."""
         self.button_widgets.clear()
 
+    def add_logo_widget(
+        self,
+        logo: Optional[Union[pyvista.ImageData, str, pathlib.Path]] = None,
+        position: Tuple[float, float] = (0.75, 0.8),
```

Comment:
I assume that `position` and `size` can actually be any `Sequence` of length 2 or `ndarray` of length 2?  Same comment for the descriptions below.

Reply:
My understanding is that when type hinting for structures that have a defined/fixed length, you need to use `List` or `Tuple` and that `Sequence` is for variable length structure

Reply:
List is a variable length container too.  What if the user gets these values from some other calculation in a different type?  This will be very annoying to change their code because we have opinionatedly required only Tuple.  This assumes they are type checking.  We should use the broadest type possible.

Reply:
> `List` is a variable length container too. What if the user gets these values from some other calculation in a different type? This will be very annoying to change their code because we have opinionatedly required only `Tuple`. This assumes they are type checking. We should use the broadest type possible.

Reply:
After a little digging, it seems using `Tuple[...]` is the correct type annotation when it is fixed length. Otherwise, I would need to add a union with `List[float]`, which would then suggest that any variable length list of floats can be passed.

Reply:
My stance is to leave this as is. If you feel strongly, please feel free to implement a correction here

Reply:
Done